### PR TITLE
RHDEVDOCS 6263 Release notes for Pipelines 1.16.1

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

cp to pipelines-docs-1.16

Issue:

RHDEVDOCS 6263 

Link to docs preview:

https://84654--ocpdocs-pr.netlify.app/openshift-pipelines/latest/release_notes/op-release-notes-1-16.html#release-notes-1-16-1_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-6263

**Review by khrm — COMMENTED** [review]
LGTM

File: modules/op-release-notes-1-16.adoc [hunk 1]
```diff
@@ -602,14 +602,34 @@ spec:
...
+== Release notes for {pipelines-title} General Availability 1.16.1
+
+With this update, {pipelines-title} General Availability (GA) 1.16.1 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-16-1_{context}"]
+=== Fixed issues
+
+* Before this update, in the *Pipelines overview* page of the web console, users that do not have access to all namespaces could select *All* in the *Projects* list. The console displayed wrong information for that selection, because the statistics for some of the namespaces were not available to the user. With this update, users who do not have access to all namespaces cannot select *All* in the *Projects* list.
+
+* Before this update, when you used the web console to start a pipeline or task that defined a parameter of the type `array`, the web console reported an error when yiou entered the value for the parameter and you could not start the pipeline or task. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for thsi parameter works normally.
```
Comment (lokanandaprabhu) [inline modules/op-release-notes-1-16.adoc:627]:
```suggestion
* Before this update, when you used the web console to start a pipeline or task that defined a parameter of the type `array`, the web console reported an error when you entered the value for the parameter and you could not start the pipeline or task. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for this parameter works normally.
```

File: modules/op-release-notes-1-16.adoc [hunk 2]
```diff
@@ -602,14 +602,34 @@ spec:
...
+
+* Before this update, when using {tekton-results}, the `tekton-results-watcher` pod sometimes crashed and a `panic: runtime error: invalid memory address or nil pointer dereference` message was logged. With this update, the `tekton-results-watcher` pod does not crash.
+
+* Before this update, when using {tekton-results}, if you enabled authentication in {tekton-results}, you could not view information from {tekton-results} in the web console, because the web console failed to pass an authentication token to the {tekton-results} API. With this update, you can view information from {tekton-results} in the web console when authentication is enabled.
+
+* Before this update, when viewing information from {tekton-results} in the web console, if the information was displayed in multiple pages, some of the information was not displayed. With this update, information from {tekton-results} displays correctly on multiple pages in the web console.
```
Comment (lokanandaprabhu) [inline modules/op-release-notes-1-16.adoc:635]:
```suggestion
* Before this update, when viewing information from {tekton-results} in the web console, if user scrolled down to the bottom, the next set of records was not fetched, causing some of the information was not displayed. With this update, as user scroll down to the bottom, records from {tekton-results} load correctly, ensuring all information displays correctly in the web console.
```

**Review by lokanandaprabhu — CHANGES_REQUESTED** [review]

**Review by ppitonak — APPROVED** [review]

**Review by lokanandaprabhu — APPROVED** [review]

File: modules/op-release-notes-1-16.adoc [hunk 3]
```diff
@@ -602,14 +602,34 @@ spec:
...
+== Release notes for {pipelines-title} General Availability 1.16.1
+
+With this update, {pipelines-title} General Availability (GA) 1.16.1 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-16-1_{context}"]
+=== Fixed issues
+
+* Before this update, in the *Pipelines overview* page of the web console, users that do not have access to all namespaces could select *All* in the *Projects* list. The console displayed wrong information for that selection, because the statistics for some of the namespaces were not available to the user. With this update, users who do not have access to all namespaces cannot select *All* in the *Projects* list.
+
+* Before this update, when you used the web console to start a pipeline or task that defined a parameter of the type `array`, the web console reported an error when you entered the value for the parameter and you could not start the pipeline or task. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for this parameter works normally.
```
Comment (abhatt-rh) [inline modules/op-release-notes-1-16.adoc:627]: took a shot at rewording it for clarity. What do you think?
```suggestion
* Before this update, you could not use the web console to start a pipeline or task that defined a parameter of the type `array` and entering a value for this parameter resulted in an error. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for this parameter works normally.
```

File: modules/op-release-notes-1-16.adoc [hunk 4]
```diff
@@ -602,14 +602,34 @@ spec:
...
+
+* Before this update, when using {tekton-results}, the `tekton-results-watcher` pod sometimes crashed and a `panic: runtime error: invalid memory address or nil pointer dereference` message was logged. With this update, the `tekton-results-watcher` pod does not crash.
+
+* Before this update, when using {tekton-results}, if you enabled authentication in {tekton-results}, you could not view information from {tekton-results} in the web console, because the web console failed to pass an authentication token to the {tekton-results} API. With this update, you can view information from {tekton-results} in the web console when authentication is enabled.
+
+* Before this update, when viewing information from {tekton-results} in the web console, when you scrolled down to the end of a page, the console failed to fetch the next set of records and some of the information was not displayed. With this update, if you scroll to the end of the page, records from {tekton-results} load correctly and all information displays correctly in the web console.
```
Comment (abhatt-rh) [inline modules/op-release-notes-1-16.adoc:635]:
```suggestion
* Before this update, when viewing information from {tekton-results} in the web console, if you scrolled down to the end of a page, the console failed to fetch the next set of records and some of the information was not displayed. With this update, if you scroll to the end of the page, records from {tekton-results} load correctly and all information displays correctly in the web console.
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @mramendi 
Mostly LGTM! Added some wording-related comments for your consideration.

Also, adding a reminder to squash your commits to one before requesting a merge review 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by adellape — APPROVED** [review]

File: modules/op-release-notes-1-16.adoc [hunk 5]
```diff
@@ -602,14 +602,34 @@ spec:
...
+== Release notes for {pipelines-title} General Availability 1.16.1
+
+With this update, {pipelines-title} General Availability (GA) 1.16.1 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-16-1_{context}"]
+=== Fixed issues
+
+* Before this update, in the *Pipelines overview* page of the web console, users that do not have access to all namespaces could select *All* in the *Projects* list. The console displayed wrong information for that selection, because the statistics for some of the namespaces were not available to the user. With this update, users who do not have access to all namespaces cannot select *All* in the *Projects* list.
+
+* Before this update, when you tried to used the web console to start a pipeline or task that defined a parameter of the type `array`, entering a value for this parameter resulted in an error and you could not start the pipeline or task. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for this parameter works normally.
```
Comment (adellape) [inline modules/op-release-notes-1-16.adoc:627]:
```suggestion
* Before this update, when you tried to use the web console to start a pipeline or task that defined a parameter of the type `array`, entering a value for this parameter resulted in an error and you could not start the pipeline or task. With this update, you can use the web console to start a pipeline or task that defines a parameter of the type `array` and entering a value for this parameter works normally.
```

**Pull request merged**